Content.Server/Objectives/Components/KillPersonConditionComponent.cs

```diff
@@ -10,8 +10,14 @@ namespace Content.Server.Objectives.Components;
 public sealed partial class KillPersonConditionComponent : Component
 {
     /// <summary>
-    /// Whether the target must be truly dead, ignores missing evac.
+    /// Whether the target must be dead
     /// </summary>
     [DataField, ViewVariables(VVAccess.ReadWrite)]
     public bool RequireDead = false;
+
+    /// <summary>
+    /// Whether the target must not be on evac
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
```

Comment:
```suggestion
    [DataField]
```
DataField incluse VV RW

Reply:
me when my reviews get ignored 😢

Reply:
man I didn't see this! I was so focused making sure the logic made sense :cry:

Reply:
Could be worse I guess.
Thanks for giving it a review 🫡